assets/pages/oneoff-contributions/components/formFields.jsx

```diff
@@ -65,24 +69,9 @@ function mapDispatchToProps(dispatch: Dispatch<UserAction | CheckoutAction>) {
 
 // ----- Component ----- //
 
-function isPayWithCardButtonFocused(event: FocusEvent): boolean {
-  const { relatedTarget } = event;
-  if (relatedTarget instanceof HTMLElement) {
-    return relatedTarget.classList.contains('component-stripe-pop-up-button');
-  }
-  return false;
-}
-
-const onBlur = (setShouldValidate: () => void) => (event: FocusEvent) => {
-  // Don't update the Redux state if the focus event is on the payment button, as this
-  // will cause a re-render and the click event on the button will be lost
-  if (!isPayWithCardButtonFocused(event)) {
-    setShouldValidate();
-  }
-};
-
-
 function NameForm(props: PropTypes) {
+  const stripeButtonClassName = 'component-stripe-pop-up-button';
```

Comment:
I know we're limited in options for catching this blur event, but we now have this `className` defined in two places 🤔. Is there another way we can identify the Stripe button?